### PR TITLE
feat(medusa): Improve exported medusa API

### PR DESCRIPTION
**Pull request by adrien2p**
Avoid to always have to import from `@medusajs/medusa/dist/*`

### CODE REVIEW

File: packages/medusa/src/models/swap.ts [hunk 1]
```diff
@@ -26,19 +26,19 @@ import { Cart } from "./cart"
...
   REQUIRES_ACTION = "requires_action",
 }
 
-export enum PaymentStatus {
+export enum SwapPaymentStatus {
   NOT_PAID = "not_paid",
   AWAITING = "awaiting",
   CAPTURED = "captured",
-  CONFIRMED = "confirmed",
+  CONFIRMED = "confqirmed",
```
[the remark this answers is not in the document]
Comment (adrien2p) [reply packages/medusa/src/models/swap.ts:41]: nicely spotted thanks @srindom

**Review by srindom — APPROVED** [review]

**Comment by srindom** [comment]
@adrien2p please look into the tests - could because this isn't up to date with the base

**Comment by adrien2p** [comment]
Yes will look into it 👌

**Comment by srindom** [comment]
@adrien2p what was the issue?

**Comment by adrien2p** [comment]
Just re run the tests @srindom 😂

**Pull request merged**